client/app/models/disposition.js

```diff
@@ -91,12 +91,16 @@ export default class DispositionModel extends Model {
   // e.g. 'Favorable', 'Conditional Favorable', 'Unfavorable', 'Conditional Unfavorable',
   // 'No Objection', 'Waiver of Recommendation', N/A is defualt
 
+
+  // TODO: These are duplicates??? Are they numbers or strings?
   @attr('number', { defaultValue: null }) dcpBoroughpresidentrecommendation;
 
   @attr('number', { defaultValue: null }) dcpBoroughboardrecommendation;
 
   @attr('number', { defaultValue: null }) dcpCommunityboardrecommendation;
 
+  @attr('number', { defaultValue: null }) dcpCitycouncilvote;
```

Comment:
hmm I'm not seeing this field used anywhere ?

Reply:
Ty, will remove. was vestigial